nas_spec/components/schemas/Payments/Payment.yaml

```
@@ -97,6 +97,12 @@ properties:
         description: Whether the payment was flagged by a risk check
         default: false
         example: true
+      score: 
+        type: integer
+        description: The risk score calculated by our Fraud detection engine. Absent if not enough data provided.
```

```suggestion
        description: The risk score calculated by our Fraud Detection engine. Absent if not enough data provided.
```
Reason for change: to align with mentions of this term in https://www.checkout.com/docs/risk-management/fraud-detection :)